src/components/teams/dialogs/JoinTeamDialog.js

```diff
@@ -0,0 +1,156 @@
+                            </Typography>
+                            <Field
+                                id={build(baseId, ids.JOIN_MSG)}
+                                name="message"
+                                label={t("joinDlgLabel")}
+                                onKeyDown={(event) => {
+                                    if (event.key === "Enter") {
+                                        handleSubmit();
+                                    }
+                                }}
+                                InputProps={{
+                                    readOnly: isLoading,
+                                    endAdornment: (
+                                        <>
+                                            {isLoading && (
+                                                <InputAdornment position="start">
+                                                    <CircularProgress
+                                                        id={build(
+                                                            baseId,
+                                                            ids.JOIN_MSG,
+                                                            ids.LOADING_SKELETON
+                                                        )}
+                                                        color="inherit"
+                                                        size={20}
+                                                    />
+                                                </InputAdornment>
+                                            )}
+                                        </>
+                                    ),
+                                }}
+                                component={FormMultilineTextField}
+                            />
+                            {joinError && (
+                                <ErrorTypographyWithDialog
+                                    errorMessage={joinError?.message}
+                                    errorObject={joinError?.object}
+                                />
+                            )}
+                        </DialogContent>
+
+                        <DialogActions>
+                            <Button
+                                id={build(baseId, ids.BUTTONS.CANCEL_BTN)}
+                                onClick={onClose}
+                            >
+                                {t("common:cancel")}
+                            </Button>
+                            <Button
+                                id={build(baseId, ids.BUTTONS.JOIN_BTN)}
+                                color="primary"
+                                type="submit"
+                                onClick={handleSubmit}
+                            >
+                                {t("join")}
+                            </Button>
+                        </DialogActions>
+                    </Dialog>
+                );
```

Comment:
Think we can use `DEDialog` here just to be consistent.

Reply:
Oops, not sure why I didn't use `DEDialog` in the first place... must've been on autopilot 😊 Thanks for catching!